apps/examples/testcase/le_tc/kernel/tc_libc_unistd.c

```diff
@@ -241,6 +242,10 @@ static void tc_libc_unistd_pipe(void)
 
 	sem_wait(&pipe_sem);
 
+	/* wait for child to exit, and store child's exit status */
+	ret_chk = wait(&status);
```

Comment:
@olegartys , Yes, I will make sure that `waitpid ` is bound with `SCHED_WAITPID`.